packages/xarc-app-dev/src/lib/utils.ts

```diff
@@ -90,27 +91,36 @@ export const jsonParser = (key, value) => {
 
 let cachedXarcOptions;
 
-export function loadXarcOptions(dir: string = process.cwd()) {
+export function loadXarcOptions(dir: string = process.cwd(), showError = true) {
+  let cwd;
+  if (process.env.XARC_CWD) {
+    cwd = process.env.XARC_CWD;
+  } else {
+    cwd = dir;
+  }
```

Comment:
line 95-100 can be achieved in line 94 like this:

`dir: string = process.env.XARC_CWD || process.cwd()`